frontend_app_admin_workspace_user/src/container/AdminWorkspaceUser.jsx

```diff
@@ -575,15 +575,6 @@ export class AdminWorkspaceUser extends React.Component {
     }))
   }
 
-  handleClickSpace = workspaceId => {
-    const { state } = this
-    if (state.workspaceIdOpened === null) {
-      state.config.history.push(PAGE.WORKSPACE.ADVANCED_DASHBOARD(workspaceId), { from: 'adminSpaceList' })
-    } else GLOBAL_dispatchEvent({ type: CUSTOM_EVENT.RELOAD_CONTENT('workspace_advanced'), data: { workspace_id: workspaceId } })
-
-    this.setState({ workspaceIdOpened: workspaceId })
```

Comment:
since `workspaceIdOpened` isn't used anymore, you can remove it's initialization in the cronstructor

Reply:
solved in [95b7c6a](https://github.com/tracim/tracim/pull/6149/commits/95b7c6a1d072be4bbd2675884c143f88cf7f3819)

Reply:
confirmed, you can resolve